File: Java Programs/Practice/PrimeCheck.java
import java.util.Scanner;

public class PrimeCheck
{
	public static void main(String args[])
	{
		Scanner sc = new Scanner(System.in);

		System.out.print("\n Enter a number : ");
		int n = sc.nextInt();

		boolean flag = true;
		for(int i = 2; i * i <= n; i++)
		{
			if(n % i == 0)
			{
				flag = false;
				break;
			}		
		}

		if(flag)
		{
			System.out.println("\n " + n + " is a prime number");
		}
		else
		{
			System.out.println("\n " + n + " is not a prime number");
		}
	}
}		

File: Python Programs/prime_check.py
n = int(input("Enter a number : "))

flag = True

for i in range(2, int(n ** 0.5) + 1):
    if(n % i == 0):
        flag = False

if(flag):
    print(n, " is a prime number")
else:
    print(n, " is not a prime number")
